libraries/stdlib/common/src/generated/_Strings.kt

```diff
@@ -55,7 +55,7 @@ public inline fun CharSequence.find(predicate: (Char) -> Boolean): Char? {
 /**
  * Returns the last character matching the given [predicate], or `null` if no such character was found.
  * 
- * @sample samples.collections.Collections.Elements.find
+ * @sample samples.text.Strings.findLast
```

Comment:
These files are generated and should be updated via `:tools:kotlin-stdlib-gen:run` command.
See https://github.com/JetBrains/kotlin/blob/master/libraries/stdlib/ReadMe.md#code-generation for details.

Reply:
I see. After I committed updates, I ran `:tools:kotlin-stdlib-gen:run`. But, after all, there was no changes of this file. Is this okay?

Reply:
Yes, it's OK. IIRC, there was a problem with the initial patch and `:tools:kotlin-stdlib-gen:run` was reverting `@sample samples.text.Strings.findLast` back to `@sample samples.collections.Collections.Elements.find`